docs/dev/earthly.rst

```diff
@@ -249,32 +211,15 @@ enumerated using ``earthly ls`` or ``earthly doc`` in the root of the repository
 
    .. seealso:: `sbom-lite` and `sbom-lite-updating`
 
+.. earthly-target:: +sbom-generate-new-serial-number
 
-.. program:: +create-silk-asset-group
-.. earthly-target:: +create-silk-asset-group
-
-   Creates a new `Silk asset group <silk-asset-group>` for a branch in the
-   repository. This target executes the `tools/create-silk-asset-group.py`
-   script with the appropriate arguments.
-
-   .. note:: For branches that execute in CI, running this target manually is
-      not necessary, as it is run automatically for every build.
-
-   .. rubric:: Parameters
-   .. option:: --branch <branch>
-
-      The repository branch for which to create the new asset group. If not
-      specified, the branch name will be inferred by asking Git.
-
-   .. rubric:: Secrets
-   .. envvar::
-         SILK_CLIENT_ID
-         SILK_CLIENT_SECRET
-      :noindex:
+   Equivalent to `+sbom-generate` but uses the `--generate-new-serial-number`
+   flag to generate a new unique serial number and reset the SBOM version to 1.
```

Comment:
I think this docs line may fail, requiring two backticks around `--generate-new-serial-number`. Recommend testing docs changes with `make -C docs/dev serve`.

Reply:
Good catch and thank you for the tip. Testing doc generation caught additional obsolete references to the old `+sbom-download` target. Inspected the generated HTMLs and updated accordingly.